packages/cdk-graph-plugin-diagram/package.json

```diff
@@ -198,6 +198,36 @@
       "rootDir": "src"
     }
   },
+  "nx": {
+    "targets": {
+      "build": {
+        "outputs": [
+          "{projectRoot}/dist",
+          "{projectRoot}/lib",
+          "{projectRoot}/build",
+          "{projectRoot}/coverage",
+          "{projectRoot}/test-reports",
+          "{projectRoot}/target",
+          "{projectRoot}/LICENSE_THIRD_PARTY",
+          "{projectRoot}/.jsii",
+          "{projectRoot}/node_modules/sharp"
+        ],
+        "dependsOn": [
+          {
+            "target": "build",
+            "projects": "dependencies"
+          }
+        ],
+        "inputs": [
+          "default",
+          "^default",
+          {
+            "runtime": "npx -p @aws-prototyping-sdk/nx-monorepo nx-dir-hasher {workspaceRoot}/packages/cdk-graph-plugin-diagram/node_modules/sharp"
```

Comment:
I assume this invalidates the caches if the nested sharp dep changes?

Reply:
Yes that is correct. It includes the hash of the `node_modules/sharp` directory in the caching hash, which if changed will re-build.

Along with the follow lines that will ensure sharp build files are output from cache as well
```ts
    this.nxOverride(
      "targets.build.outputs",
      ["{projectRoot}/node_modules/sharp"],
      true
    );
```